manifests/c/CHIRP/CHIRP/daily-20210814/CHIRP.CHIRP.yaml

```diff
@@ -0,0 +1,16 @@
+PackageIdentifier: CHIRP.CHIRP
+PackageVersion: daily-20210814
+PackageLocale: en-US
+Publisher: CHIRP
+PackageName: CHIRP
+License: GPL-3.0
+ShortDescription: A free, open-source tool for programming your amateur radio / ham radio transceiver.
+Installers: 
+ - Architecture: x86
+   InstallerType: nullsoft
+#   NOTE - when updating this, do NOT use the /LATEST? path; use the specific date subdirectpory
+   InstallerUrl: https://trac.chirp.danplanet.com/chirp_daily/daily-20210814/chirp-daily-20210814-installer.exe
+   #InstallerSha256: 
+   InstallerSha256: 7ac5f2b3871d69c01d19720ada431c3863663283436df4e66df944cdd30e2f0b
```

Comment:
```suggestion
   InstallerSha256: 7ac5f2b3871d69c01d19720ada431c3863663283436df4e66df944cdd30e2f0b
   ProductCode: CHIRP
```

Reply:
since it doesn't show a version and publisher in Control Panel, we need to add this ProductCode:
![image](https://user-images.githubusercontent.com/11600822/129479936-478b0a8e-9d37-42e9-bc00-151ad13a0437.png)